modules/rostests/apitests/ntdll/UserModeException.c

```diff
@@ -64,7 +64,7 @@ DetermineCpuVendor(void)
     Vendor.ShuffledInts[1] = CpuInfo[3];
     Vendor.ShuffledInts[2] = CpuInfo[2];
     Vendor.VendorString[12] = 0;
-
+    fprintf(stderr, "Vendor: %s\n", Vendor.VendorString);
```

Comment:
Is the `trace()` macro available? Or did you use fprintf because the former was unavailable or did unwanted behaviour?

Reply:
I changed it to trace now.